.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+
+<!-- 
+
+Sorry to hear you're having trouble! If you have a question, please ask in 
+[SO or gitter](https://github.com/callemall/material-ui/blob/master/CONTRIBUTING.md#asking-questions)
+
+If you are having an issue with click events, please re-read the 
+[README](https://github.com/callemall/material-ui#react-tap-event-plugin) (you did read the README, right?).
+
+Questions, or issues that don't provide sufficient information, may be closed without comment. 
+(We're sure you'd rather we work on improving Material-UI than chasing ghost issues.)
+
+If you think you have found a _new_ issue that hasn't already been reported, please complete the template below.
+
+-->
+
+Problem Description
+===================
+<!-- Please include steps to reproduce the problem, code sample, screenshots as appropriate. -->
+
+Versions
+========
+<!-- Versions affected. (Please test with Material-UI HEAD.) -->
+- Material-UI:
+- React:
+- Browser:
+- OS:
```

Comment:
Maybe we could skip OS? I've rarely see that to be the cause of the problem unless it's build related, but in that case the OS version including NPM version would be helpful. I'd say let's remove it and if we find we're asking for OS too in a lot of issues we can add it back.